Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -33,6 +33,10 @@
             {%- endfor %}
         {%- endif %}
 
+        {%- if item.extra('route') is not empty and item.extra('route') == request.get('_route') %}
+            {%- set active = true %}
+        {%- endif %}
+
```

Comment:
> @tim96 this looks like it should be an MenuVoter (model) instead of being implemented in the view.

@mvhirsch Why it should be in model? You can see previous line in this block, active menu always set in this function, in the view.